cookiecutter/generate.py

```diff
@@ -163,6 +167,11 @@ def render_and_create_dir(dirname, context, output_dir):
     dir_to_create = os.path.normpath(
         os.path.join(output_dir, rendered_dirname)
     )
+    if os.path.exists(dir_to_create):
+        if fail_if_exists:
```

Comment:
Let's swap these if clauses. We do not need to check for existence of the dir for `fail_if_exists == False`.